From 92779d46577370da9750a36a560a4a983c71cbe9 Mon Sep 17 00:00:00 2001
From: Erez Shaul <eshaul@habana.ai>
Date: Wed, 06 Nov 2024 08:11:54 +0200
Subject: [PATCH] [SW-206938] hl-smi: change hl-smi-telem tool name

Tool name changed from hl-smi-telem to hl-smi-async

Change-Id: I2f876c20ce1486c38acef3f53a1f6e9593d6ba75
Signed-off-by: Erez Shaul <eshaul@habana.ai>
---

diff --git a/package/hl-smi/hl-smi.mk b/package/hl-smi/hl-smi.mk
index 8e86b99..1aafdca 100644
--- a/package/hl-smi/hl-smi.mk
+++ b/package/hl-smi/hl-smi.mk
@@ -9,7 +9,7 @@
 
 define HOST_HL_SMI_INSTALL_CMDS
 	$(INSTALL) -s -D -m 0755 $(@D)/bin/hl-smi $(BINARIES_DIR)/hl-smi
-	$(INSTALL) -s -D -m 0755 $(@D)/bin/hl-smi-telem $(BINARIES_DIR)/hl-smi-telem
+	$(INSTALL) -s -D -m 0755 $(@D)/bin/hl-smi-async $(BINARIES_DIR)/hl-smi-async
 	$(INSTALL) -D -m 0644 $(@D)/lib/libhlml.a $(HOST_DIR)/lib/libhlml.a
 	$(INSTALL) -D -m 0644 $(@D)/lib/libhlml.so $(HOST_DIR)/lib/libhlml.so
 	$(INSTALL) -D -m 0644 $(SPECS_EMBEDDED_DIR)/hlml.h $(HOST_DIR)/include/hlml.h
